pkgs/tools/system/osquery/default.nix

```diff
@@ -0,0 +1,79 @@
+{ pkgs }:
+
+with pkgs;
+let
+  buildStdenv = overrideCC stdenv llvmPackages.clangUseLLVM;
+  opensslArchive =
+    let
+      # https://github.com/osquery/osquery/blob/master/libraries/cmake/formula/openssl/CMakeLists.txt#L3-L4
```

Comment:
Can you make this a permalink? (And throughout)

https://github.com/osquery/osquery/blob/d8330d915869281a366eb8f48c42a3a7a14e418f/libraries/cmake/formula/openssl/CMakeLists.txt#L3-L4

Reply:
Fixed in 154c57bfd89.

I've opted to use commit `877d5e69ab97e15800b5c5128b3de2cf6f322942` as it is the one tagged with `5.5.1` as shown by `git rev-parse`.

When we go to update the package, I guess we replace the commit with the tag or the revision the tag currently points at and double check the necessity of the build inputs and validity of these comments